e2e/e2e_test.go

```diff
@@ -125,6 +142,26 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	var config rest.Config
 	Expect(dec.Decode(&config)).To(Succeed())
 	restConfig = &config
+	e2eutil.RestConfig = &config
+	dc, err := discovery.NewDiscoveryClientForConfig(restConfig)
+	Expect(err).To(Succeed())
+	mapper := restmapper.NewDeferredDiscoveryRESTMapper(memory.NewMemCacheClient(dc))
+
+	// Databases (except spanner) are spun up once per test suite run,
```

Comment:
why isn't spanner not treated like the other databases?

Reply:
the spanner emulator doesn't support concurrent requests, so we have to spin up a separate spanner pod per test 😢 

> The emulator only allows one read-write transaction or schema change at a time. Any concurrent transaction will be aborted. Transactions should always be wrapped in a retry loop. 

From: https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/blob/master/README.md